src/Core/BackendAggregator.vala

```diff
@@ -24,6 +24,28 @@ public class AppCenterCore.BackendAggregator : Backend, Object {
         backends = new Gee.ArrayList<unowned Backend> ();
         backends.add (PackageKitBackend.get_default ());
         backends.add (UbuntuDriversBackend.get_default ());
+        backends.add (FlatpakBackend.get_default ());
+
+        foreach (var backend in backends) {
+            backend.notify["working"].connect (() => {
+                // Trigger a notify on this class, value doesn't matter
+                working = true;
```

Comment:
Wouldn't it make more sense?
```suggestion
                notify_property ("working");
```

Reply:
It would indeed. Every day is a school day :wink: 

I was trying to figure out how `notify["working"] ()` worked and gave up without realising that `notify_property` exists :slightly_smiling_face: